pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -562,6 +549,29 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 	}
 	log.Noticef("Have %d assignable adapters", len(aa.IoBundleList))
 
+	if err := utils.WaitForVault(ps, log, agentName, warningTime, errorTime); err != nil {
+		log.Fatal(err)
+	}
+
+	log.Functionf("processed vault status")
+
+	if err := containerd.StartUserContainerdInstance(); err != nil {
```

Comment:
Does this mean that it is `domainmgr` that starts the user containerd? Is that the right place to start it?

Reply:
Any suggestions? I received https://github.com/lf-edge/eve/pull/2524#discussion_r811426341